roles/edpm_extra_mounts/defaults/main.yml

```diff
@@ -23,5 +23,5 @@ edpm_extra_mounts_hide_sensitive_logs: true
 
 edpm_extra_mounts: []
 edpm_default_mounts: [
-  {name: 'hugepages1G', path: '/dev/hugepages1G', opts: 'pagesize=1G', fstype: 'hugetlbfs'},
-  {name: 'hugepages2M', path: /dev/hugepages2M, opts: 'pagesize=2M', fstype: 'hugetlbfs'}]
+  {name: 'hugepages1G', path: '/dev/hugepages1G', opts: 'pagesize=1G', fstype: 'hugetlbfs', group: 'hugetlbfs'},
+  {name: 'hugepages2M', path: /dev/hugepages2M, opts: 'pagesize=2M', fstype: 'hugetlbfs', group: 'hugetlbfs'},]
```

Comment:
ansilble-lint is failing because of this trailing comma.

Reply:
thank you!